### PR TITLE
Yro/update logging

### PR DESCRIPTION
**Pull request by yro**
### Description
Updates logging. Standardizes formatting.

### Testing
- [x] Unit tests

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @schenedx 

FYI: @edx/educator-neem

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

File: control/veda_deliver_youtube.py [hunk 1]
```diff
@@ -190,50 +198,42 @@ def batch_uploader(self):
...
+                        'youtube_callback',
+                        'static_files',
+                        'delivery.complete'
+                    ),
+                    callback=printTotals,
+                    confirm=False,
+                    preserve_mtime=False
+                )
+        except paramiko.ssh_exception.AuthenticationException:
+            LOGGER.info('[YOUTUBE] {file} : Paramiko Authentication Exception'.format(file=str(self.file)))
```
Comment (yro) [inline control/veda_deliver_youtube.py]: This is a little messy, but it's our highest error on NR.

File: VEDA_OS01/fixtures/encodes.json [hunk 2]
```diff
@@ -160,5 +160,18 @@
...
             "product_spec": "desktop_webm"
         }
+    },
+    {
+        "model": "VEDA_OS01.course",
+        "pk": 1,
+        "fields": {
+            "course_name": "Demo Course",
+            "institution": "XXX",
+            "edx_classid": "XXXXX",
```
Comment (schenedx) [inline VEDA_OS01/fixtures/encodes.json]: #NIT instead of `xxxxx`, can these fake value be something easier to read?

File: control/veda_utils.py [hunk 3]
```diff
@@ -109,7 +92,7 @@ def status_bar(process):
         sys.stdout.flush()
 
 
-class Report():
+class Report:
```
Comment (schenedx) [inline control/veda_utils.py]: Why not just do `class Report(object):`

File: VEDA_OS01/fixtures/encodes.json (hunk 2 above)
Comment (yro) [reply VEDA_OS01/fixtures/encodes.json]: This should be an 3/5 letter code that gets concatenated to (for example) MIT6002X, so it shouldn't be super easy to read, and ideally won't collide with another course (if added).
I'll do EDXDEMOX.

File: control/veda_utils.py (hunk 3 above)
Comment (yro) [reply control/veda_utils.py]: 💯

**Comment by yro** [comment]
@schenedx Feedback to date addressed.

File: VEDA_OS01/fixtures/encodes.json [hunk 4]
```diff
@@ -160,5 +160,18 @@
...
+        "model": "VEDA_OS01.course",
+        "pk": 1,
+        "fields": {
+            "course_name": "Demo Course",
+            "institution": "EDX",
+            "edx_classid": "DEMOX",
+            "semesterid": 2018,
+            "yt_proc": false,
+            "local_storedir": "this/is/an/exemplar",
+            "studio_hex": "xxxx"
```
Comment (schenedx) [inline VEDA_OS01/fixtures/encodes.json]: #NIT Just one more here to make it easier to read

File: control/veda_deliver_youtube.py [hunk 5]
```diff
@@ -1,32 +1,41 @@
...
-        print 'Callback Failing'
-    """
-    return None
+        LOGGER.error('Callback Failing')
+    '''
+    return
 
 
-class DeliverYoutube():
+class DeliverYoutube:
```
Comment (schenedx) [inline control/veda_deliver_youtube.py]: Sorry, missed this place. Could you make it ‘class DeliverYoutube(object):’

Comment (yro) [reply control/veda_deliver_youtube.py]: I went through and cleaned up the others as well -- there's some places where we can probably use inheritance as well, but in the interest of delivering I'll end my refactorfest 😄

**Comment by yro** [comment]
Thanks @schenedx

**Pull request merged**